.github/actions/combine-results/action.yaml

```diff
@@ -21,11 +21,11 @@ runs:
       echo "Summary:"
       failed="false"
       while IFS=" " read -r job result; do
-        if [[ "$result" != "success" ]]; then
+        if [[ "$result" == "success" ]] || [[ "$result" == "skipped" ]]; then
```

Comment:
for reference, it may be useful to print out all the statuses while we're doing changes here, and treat canceled/failed as actual failures

Reply:
Done. I added a comment. Anything but "success" and "skipped" will be failures and will fail that job. This way, we can catch any future addition of statuses by github. Right now, only "cancelled" and "failure" will cause this step to fail, but any new future status will also be a failure.